src/app/projects/ProjectPlan.tsx

```diff
@@ -89,7 +79,7 @@ function ProjectPlan(props: IProjectPlanProps): JSX.Element {
                     <ToolbarContent>
                         <ToolbarItem>
                             <Button
-                                isDisabled={disableThisSection}
+                                isDisabled={isPlanning}
```

Comment:
Keep the variable name as `disableThisSection` and just define it as `const disableThisSection = isPlanning;` somewhere above.